include/mbedtls/cmac.h

```diff
@@ -93,8 +94,9 @@ int mbedtls_cipher_cmac_starts( mbedtls_cipher_context_t *ctx,
  * \param input         The buffer holding the input data.
  * \param ilen          The length of the input data.
  *
- * \returns             \c 0 on success, or #MBEDTLS_ERR_MD_BAD_INPUT_DATA
- *                      if parameter verification fails.
+ * \returns            \c 0 on success.
+ * \returns            #MBEDTLS_ERR_MD_BAD_INPUT_DATA
```

Comment:
Above, the `MBEDTLS_XXX` constants are not prefixed by `#`; we should make this uniform.

Reply:
# is a linking mechanism. The _XXX error codes are a generalization, and therefore cannot be linked. N/A